src/Symfony/Bundle/FrameworkBundle/Command/ServerStartCommand.php

```diff
@@ -244,4 +249,20 @@ private function createServerProcess(OutputInterface $output, $address, $documen
 
         return new Process('exec '.$script, $documentRoot, null, null, null);
     }
+
+    private function launchBrowser($output, $address)
+    {
+        $cmdOpen = 'xdg-open';
+        if (stristr(PHP_OS, 'WIN')) {
```

Comment:
for consistency with the rest of the Symfony codebase I would use `stripos` here